src/app/content/studyGuides/components/Filters.tsx

```diff
@@ -31,6 +34,14 @@ const StyledColorFilter = styled(ColorFilter)`
   min-width: 29rem;
 `;
 
+// tslint:disable-next-line: variable-name
+const RightButtonsWrapper = styled.div`
+  margin-left: auto;
+  display: flex;
+  align-items: center;
+  height: ${barHeight}rem;
```

Comment:
This should be different for the mobile version, however I think it is not necessary to use static height

Reply:
This is causing chapter / color filter to not be aligned to the top on mobile resolution:
![image](https://user-images.githubusercontent.com/31478212/90631205-97b1d180-e222-11ea-8d96-27b81b766ca2.png)